components/constants.go

```diff
@@ -5,6 +5,7 @@ import "github.com/jorgerojas26/lazysql/app"
 var App = app.App
 
 const (
-	EditorTabName string = "Editor"
-	HelpPageName  string = "Help"
+	EditorTabName   string = "Editor"
+	HelpPageName    string = "Help"
+	SidebarPageName string = "Sidebar"
```

Comment:
Unless you think your constant are reused externally, they should be unexported.

So this would be better

```suggestion
	editorTabName   string = "Editor"
	helpPageName    string = "Help"
	sidebarPageName string = "Sidebar"
```

also usually constant of the same kind share a common prefix, not suffix. this way you can use the first letter to find them all in your IDE

So this would be what I would use


```suggestion
	tabNameEditor   string = "Editor"
	pageNameHelp    string = "Help"
	pageNameSidebar string = "Sidebar"
```